apps/src/lab2/types.ts

```diff
@@ -96,6 +96,9 @@ export interface LevelProperties {
   levelData?: LevelData;
   appName: AppName;
   longInstructions?: string;
+  systemPrompt?: string;
+  botTitle?: string;
+  botDescription?: string;
```

Comment:
Can we make a separate `AiChatLevelProperties` which `extends LevelProperties` inside of aichat/types? That way we don't have to keep adding lab-specific types to this interface.

Reply:
Great. idea! Thanks!